server/routes/v1/upload.ts

```diff
@@ -201,6 +218,32 @@ export const postUpload = [
       res.json({
         uuid: model.uuid,
       })
+
+      if (buildOptions.rawModelExport) {
```

Comment:
I think we should keep a copy of the raw model / code here regardless of whether `rawModelExport` is enabled.  So, just run this code regardless.

Reply:
Sure - I made this condition because I thought it would save on storage etc if the user doesn't want the files stored. Happy to have them stored anyway, though. 

Maybe not for this ticket but if we're always storing the files the user could perhaps edit the version and allow for uses to export the model code even if the original version upload didn't tick the box.